index.html

```diff
@@ -0,0 +1,161 @@
+                    <li class="mobile-links"><a class="mobile-link-titles" href="" id="company-link">GASTONOMY EVENTS</a></li>
+                </ul>
+            </nav>
+
+        </header>
+
+        <section class="home-intro">
+            <h2 class="call-home">Gather All the Cooks!</h2>
+            <h1>GASTRONOMY EVENTS CAMP RETREAT <br> 2022</h1>
+            <p class="intro">Pack all your knives and join us at our lovely camp to explore, relax, have fun and of
+                course, to share the joy of cooking.</p>
+            <h3 class="date">2022.05.21(SAT) ~ 21(SUN)</h3>
+            <p class="location">@ Camp Turtle, Turgutreis, Bodrum</p>
+        </section>
+    </main>
+
+    <section class="program">
+        <h2 class="program-heading">Main Program</h2>
+        <hr class="red-hr">
+        <div class="slots">
+            <div class="slot">
+                <img class="icon" src="images/icons/spices.png" alt="">
+                <h3 class="topic">Spices</h3>
+                <p class="program-info">Learn about famuous and rare spices of the area.</p>
+            </div>
+
+            <div class="slot">
+                <img class="icon" src="images/icons/tofu.png" alt="">
+                <h3 class="topic">Tofu</h3>
+                <p class="program-info">Discover the rich posibilities of tofu.</p>
+            </div>
+
+            <div class="slot">
+                <img class="icon" src="images/icons/tools.png" alt="">
+                <h3 class="topic">Tools</h3>
+                <p class="program-info">Explore country's traditional kitchenware.</p>
+            </div>
+
+            <div class="slot">
+                <img class="icon" src="images/icons/recipe.png" alt="">
+                <h3 class="topic">Recipe</h3>
+                <p class="program-info">Share your favorites and hacks with fellow cooks.</p>
+            </div>
+
+            <div class="slot">
+                <img class="icon" src="images/icons/picnic.png" alt="">
+                <h3 class="topic">Picnic</h3>
+                <p class="program-info">Best way to celebrate cooking is to eat.</p>
+            </div>
+        </div>
+
+        <button class="join mobile">Make your reservation</button>
+
+        <a class="program-link desktop">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section class="speakers">
+
+        <h2 class="speaker-heading">Featured Speakers</h2>
+
+        <hr>
+
+        <div class="speakers-container">
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+        </div>
+
+        <!-- <button class="speaker-btn more">More <img class="down" src="images/down.png" alt=""></button>
+
+            <button class="speaker-btn less">Less <img class="down" src="images/up.png" alt=""></button> -->
```

Comment:
- [ ] Kindly delete commented code from your project, you can leave comments about code but not commented code 😉